versioned_docs/version-5.x/hello-react-navigation.md

```diff
@@ -59,7 +59,7 @@ function App() {
 export default App;
 ```
 
-![Basic app using stack navigator](/assets/navigators/stack/basic_stack_nav.png)
+<img src="/assets/7.x/fundamentals/basicAppUsingStackNavigator.jpeg" width="300" alt="Basic app using stack navigator" />
```

Comment:
Why don't we refer to `/assets/navigators` anymore and instead we rely on versioned (I guess?) directory?